03-other-topics/003-quick-reference.md

```diff
@@ -0,0 +1,50 @@
+---
+category: cloud-platform
+expires: 2019-01-01
+---
+# Kubernetes quick reference
+
+This document acts as a quick reference to `kubectl`, listing some of the most common operations.
+
+The examples here only address the most basic approach to these operations. For more options, please refer to the command-line help of `kubectl` subcommands.
+
+There is also a more detailed [cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) in the official kubernetes documentation.
+
+## Table of contents
+{:toc}
```

Comment:
Yeah this doesnt work. You can add any aribtrary list immediately before it to make it work:

```
* TOC
{:toc}
```

That works.

*However* you also get a table of contents autogenerated in the left panel of the page, so do you need two of them?

<img width="1364" alt="screen shot 2018-07-12 at 16 52 50" src="https://user-images.githubusercontent.com/3952074/42644658-090d2560-85f4-11e8-988c-68dbb8727257.png">

Reply:
...and as I can now see from looking at the screenshot (isn't it weird how looking at a screenshot is so different to looking at your _screen_?) the table of contents that is generated contains the page title and `table of contents` as a heading. You can remove this by adding a `no_toc` class to the heading, which in `Kramdown` you do like the following:

```
# Kubernetes quick reference
{:.no_toc}

```

⬆️ that will stop "Kubernetes quick reference" showing up.

Reply:
Actually, turns out that in turn breaks the way the side contents works 
<img width="1364" alt="screen shot 2018-07-12 at 17 03 16" src="https://user-images.githubusercontent.com/3952074/42645254-7e4dae98-85f5-11e8-8d77-c7d79b7fbdb1.png">

So I'm even less convinced we need the contents inline (I don't know how to fix this problem!)